### ISSUE TITLE
App does not load Flash content on Windows 7 SP1 32-bit

### ISSUE THREAD

**Issue by wesinator** [ruffle-rs/ruffle#9073]
### Describe the bug

Ruffle does not load flash content when running the nightly build of the desktop app on Windows 7 SP1 32-bit/x86.
The file picker dialog pops up to select a .swf, but once a [supported swf](https://github.com/ruffle-rs/ruffle/wiki/Test-SWFs) is selected, the program closes without running the flash content. 

**to reproduce:**
 - Windows 7 SP1 32-bit OS (e.g. test in a VM)
 - Download and extract ruffle nightly build .exe
 - Install [this update](http://go.microsoft.com/fwlink/?LinkId=848159) to ensure direct3d dll d3dcompiler_47.dll is installed. If you don't do this step you get an error when opening the program about d3dcompiler_47.dll missing. 
 - Now open the program, the dialog should pop up. select a supported swf (I tested with [Bitey of Brackenwood](https://ruffle.rs/demo/swfs/bitey1.swf) "bitey1.swf". It was the same result with other swf's that were tested).

After you select the swf file and click the 'open' button, nothing happens. 

### Expected behavior

On Windows 7 SP1 32-bit, selecting a swf file to load should load the flash content.

Are there additional dependencies to run on Windows 7 that need to be documented/installed ? 

### Affected platform

Desktop app

### Operating system

Windows 7 SP1 x86

### Additional information

![ruffle nightly 2023-01-08 dialog on windows 7 sp1 - "load a flash file"](https://user-images.githubusercontent.com/5124946/211211564-d4ae61ea-37fd-4461-bb90-2b3f5a53177b.png)

**Comment by n0samu** [comment]
The Ruffle desktop app requires hardware acceleration to run properly, which VM software generally does not provide. Please test again on a real machine if possible, otherwise I suggest closing the issue.

**Comment by mero09-99** [comment]
yeah, there are no 'additional dependencies that need to be documented' its already said you need hardware acceleration
use real hardware.
